test/googledrive/run.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+set -e
+
+echo "DEBUG: Script started"
```

Comment:
It'd be good to remove all the logging from here once this is done, but I guess that's what you planned anyways.